packs/slack/sensors/slack_sensor.py

```diff
@@ -24,6 +25,11 @@ def __init__(self, sensor_service, config=None, poll_interval=None):
                                           poll_interval=poll_interval)
         self._logger = self._sensor_service.get_logger(__name__)
         self._token = self._config['sensor']['token']
+        if 'strip_formatting' in self._config['sensor']:
```

Comment:
Please also add documentation for this new config option to the readme.

Besides that, LGTM.